plutus-pab/tx-inject/Main.hs

```diff
@@ -0,0 +1,187 @@
+{-# LANGUAGE DeriveAnyClass     #-}
+{-# LANGUAGE NamedFieldPuns     #-}
+{-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE OverloadedStrings  #-}
+{-# LANGUAGE StrictData         #-}
+
+module Main where
```

Comment:
This needs some comments
* What does the module do
* What is `Stats`
* top-level functions should have haddocks